### PR TITLE
Add help messages for COPY, ECHO and RD/RMDIR commands

### PR DESCRIPTION
**Pull request by Wengier**
This PR adds full help messages for DOS commands including COPY, ECHO, and RD/RMDIR, using the style of commands such as MOUNT and VER.

### CODE REVIEW

File: src/shell/shell.cpp [hunk 1]
```diff
@@ -840,7 +868,25 @@ void SHELL_Init() {
...
+	        "  \033[32;1mcopy\033[0m \033[37;1mSOURCE\033[0m \033[36;1m[DESTIONATION]\033[0m\n"
+	        "  \033[32;1mcopy\033[0m \033[37;1mSOURCE1+SOURCE2[+...]\033[0m \033[36;1m[DESTIONATION]\033[0m\n"
+	        "\n"
+	        "Where:\n"
+	        "  \033[37;1mSOURCE\033[0m       can be either an exact filename or an inexact filename with\n"
+	        "               wildcards, which are the asterisk (*) and the question mark (?).\n"
+	        "  \033[36;1mDESTIONATION\033[0m An exact filename or directory, not containing any wildcards.\n"
+	        "\n"
+	        "Notes:\n"
+	        "  By using \033[37;1m+\033[0m operator, you can combine multiple source files to a single file.\n"
```
Comment (kcgen) [inline src/shell/shell.cpp:883]: Add `the` before `+`.

(or an alternative; although I like yours too - your call!):
` - The + operator combines multiple source files into a single destination file.`

File: src/shell/shell.cpp [hunk 2]
```diff
@@ -840,7 +868,25 @@ void SHELL_Init() {
...
+	        "  If no destination provided, source files will be copied to the current path.\n"
```
Comment (kcgen) [inline src/shell/shell.cpp:884]: A possible alternative:

` - Destination is optional: if omitted, the files are copied to the current path.`

**Review by kcgen — CHANGES_REQUESTED** [review]
Thanks for another nice PR, @Wengier !

Just a couple minor comments.

File: src/shell/shell.cpp [hunk 3]
```diff
@@ -781,17 +781,45 @@ void SHELL_Init() {
...
+	        "  \033[32;1mecho\033[0m \033[36;1m[on|off]\033[0m\n"
+	        "  \033[32;1mecho\033[0m \033[36;1m[MESSAGE]\033[0m\n"
+	        "\n"
+	        "Where:\n"
+	        "  \033[36;1mon|off\033[0m  Turns on/off command echoing.\n"
+	        "  \033[36;1mMESSAGE\033[0m The message to display.\n"
+	        "\n"
+	        "Notes:\n"
+	        "  - The command without a parameter displays current command echoing status.\n"
+	        "  - This command is especially useful for batch files.\n"
```
[the remark this answers is not in the document]
Comment (Wengier) [reply src/shell/shell.cpp]: Done. Thanks.

File: src/shell/shell.cpp [hunk 4]
```diff
@@ -840,7 +868,25 @@ void SHELL_Init() {
 	        "  \033[32;1mdel\033[0m \033[36;1mtest.bat\033[0m\n"
 	        "  \033[32;1mdel\033[0m \033[36;1mc*.*\033[0m\n"
 	        "  \033[32;1mdel\033[0m \033[36;1ma?b.c*\033[0m\n");
-	MSG_Add("SHELL_CMD_COPY_HELP","Copy files.\n");
+	MSG_Add("SHELL_CMD_COPY_HELP", "Copies one or more files.\n");
+	MSG_Add("SHELL_CMD_COPY_HELP_LONG",
+	        "Usage:\n"
+	        "  \033[32;1mcopy\033[0m \033[37;1mSOURCE\033[0m \033[36;1m[DESTIONATION]\033[0m\n"
```
[the remark this answers is not in the document]
Comment (Wengier) [reply src/shell/shell.cpp]: Fixed.

File: src/shell/shell.cpp [hunk 5]
```diff
@@ -840,7 +868,25 @@ void SHELL_Init() {
...
+	        "  \033[32;1mcopy\033[0m \033[37;1mSOURCE1+SOURCE2[+...]\033[0m \033[36;1m[DESTIONATION]\033[0m\n"
+	        "\n"
+	        "Where:\n"
+	        "  \033[37;1mSOURCE\033[0m       can be either an exact filename or an inexact filename with\n"
+	        "               wildcards, which are the asterisk (*) and the question mark (?).\n"
```
[the remark this answers is not in the document]
Comment (Wengier) [reply src/shell/shell.cpp]: Done

File: src/shell/shell.cpp (hunk 1 above)
Comment (Wengier) [reply src/shell/shell.cpp]: Adding "the" directly will make the line exceeds 80 characters, but the alternative will work.

File: src/shell/shell.cpp (hunk 2 above)
Comment (Wengier) [reply src/shell/shell.cpp]: Done.

**Comment by Wengier** [comment]
@kcgen I have fixed all of them according to your comments. Thanks for the review.

**Comment by kcgen** [comment]
Thank you, @Wengier !

**Pull request merged**